docs/tutorials/essentials/part-1-overview-concepts.md

```diff
@@ -409,26 +409,26 @@ console.log(currentValue)
 
 ### Redux Application Data Flow
 
-Earlier, we talked about "one-way data flow", which describes this sequence of steps to update the app:
+Earlier, we talked about "one-way data flow", which describes this sequence of steps to update the app.  These steps are:
 
-- State describes the condition of the app at a specific point in time
-- The UI is rendered based on that state
-- When something happens (such as a user clicking a button), the state is updated based on what occurred
-- The UI re-renders based on the new state
+1. State describes the condition of the app at a specific point in time.
+2. The UI is rendered based on that state.
+3. When something happens (such as a user clicking a button), the state is updated (based on what occurred).
+4. The UI re-renders based on the new state.
+
+For Redux specifically, we can break these steps out into more detail:
 
-For Redux specifically, we can break these steps into more detail:
+**Initial Setup**
+1. A Redux store is created using a root reducer function.
+2. The store calls the root reducer once and saves the return value as its initial `state`.
+3. When the UI is first rendered, UI components access the current state of the Redux store and use that data to decide what to render. The UI components also subscribe to any future store updates because they want to know if the state has changed.
 
-- Initial setup:
-  - A Redux store is created using a root reducer function
-  - The store calls the root reducer once, and saves the return value as its initial `state`
-  - When the UI is first rendered, UI components access the current state of the Redux store, and use that data to decide what to render. They also subscribe to any future store updates so they can know if the state has changed.
-- Updates:
-  - Something happens in the app, such as a user clicking a button
-  - The app code dispatches an action to the Redux store, like `dispatch({type: 'counter/increment'})`
-  - The store runs the reducer function again with the previous `state` and the current `action`, and saves the return value as the new `state`
-  - The store notifies all parts of the UI that are subscribed that the store has been updated
-  - Each UI component that needs data from the store checks to see if the parts of the state they need have changed.
-  - Each component that sees its data has changed forces a re-render with the new data, so it can update what's shown on the screen
+**Updates**
+1. When something happens in the app (such as a user clicking a button), the app code dispatches an action to the Redux store (like `dispatch({type: 'counter/increment'})`).
+2. The store runs the reducer function again with the previous `state` and the current `action`.  It then saves the return value as the new `state`.
```

Comment:
This should be a compound sentence, otherwise you'd need to make it another step (which it's not)

```suggestion
2. The store runs the reducer function again with the previous `state` and the current `action`, and then saves the return value as the new `state`.
```